crates/rpc/src/utils.rs

```diff
@@ -0,0 +1,28 @@
+use alloy::providers::Provider as _;
+use ethui_types::prelude::*;
+
+pub async fn get_code(address: Address, chain_id: u32) -> jsonrpc_core::Result<Option<String>> {
```

Comment:
this doesn't need to returna `jsonrpc_core::Result`. can be a `color_eyre::Result` which is our default

also, instead of `String`, it should return `Bytes`